exampleDataTrack/src/main/java/com/twilio/video/examples/datatrack/DataTrackActivity.java

```diff
@@ -320,6 +320,23 @@ public void onRecordingStarted(Room room) {
             public void onRecordingStopped(Room room) {
 
             }
+
+            @Override
+            public void onReconnecting(Room room, TwilioException exception) {
+                Toast.makeText(DataTrackActivity.this,
+                        String.format("Reconnecting to room %s, exception = %s", room.getName(),
```

Comment:
Would you be open to showing a permanent view during the reconnecting event? And I thought we were using snackbar these days.

Reply:
Show in `onReconnecting` and dismiss in `onDisconnected` or `onReconnected`

Reply:
Yeah I can borrow the UI from the other modules.

Reply:
Pretty sure this is how `whatsapp` and `messenger` work

Reply:
@idelgado updated all modules to include reconnecting UI